test/integration/test-example-validation.js

```diff
@@ -30,7 +30,7 @@ if (!window.validatorLoad) {
   })();
 }
 
-describe.configure().retryOnSaucelabs().run('example', function() {
+describe.skip('example', function() {
```

Comment:
Why skip all of these? Can we fix the failing example to validate or remove it from the list? If flaky on SauceLabs, I think a `.skipSauceLabs()` would be great to have especially for tests like this that don't really need cross-browser testing. ( as far as I remember, we run all tests including integration on Travis using latest Chrome in addition to running integration on SauceLabs, @rsimha-amp  is still still the case > )